crowdfunding_compassion/views/project_donation_form_page.xml

```diff
@@ -0,0 +1,154 @@
+                                                <h3 class="blue">CHF</h3>
+                                            </div>
+                                        </label>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <!-- Project card -->
+                            <div class="col-sm-12 col-md-4">
+                                <t t-set="display_impact" t-value="True" />
+                                <t t-call="crowdfunding_compassion.project_impact_card" />
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <div class="container mb-5">
+                        <h3 class="blue uppercase">Your billing information</h3>
+                        <div t-attf-class="{{form.form_wrapper_css_klass}}">
+                            <t t-raw="form.form_render()"/>
+                        </div>
+                    </div>
+                </section>
+            </t>
+
+            <script>
+                const amountField = document.getElementById('custom-amount-field');
+                const amountCard = document.getElementById('custom-amount-card');
+                const amountHiddenField = document.getElementById('amount');
+                const submit = document.querySelector('button[type="submit"]');
+                const radios = document.querySelectorAll('input[type="radio"]');
+
+                // Select custom amount card when its textfield is clicked
+                amountField.addEventListener('click', () => {
+                amountCard.checked = true;
+                clickHandler();
+                });
+
+                // Update the value of the card on textfield input
+                amountField.addEventListener('input', () => { amountCard.value = amountField.value; });
+
+                // Set value of selected amount to hidden amount form field
+                submit.addEventListener('click', () => {
+                amountHiddenField.value = document.querySelector('input[name="amount"]:checked').value;
+                })
+
+                // Disable button unless an amount is selected
+                function clickHandler() {
+                let checked_radios = document.querySelectorAll('input[type=radio]:checked').length;
+                submit.disabled = checked_radios == 0;
+                }
+
+                // Register on each radio change
+                radios.forEach(radio => {radio.addEventListener('click', clickHandler);});
+
+                // Exectute on page load
+                clickHandler();
+            </script>
+        </template>
+
+        <!-- Add Terms and Conditions to donation form -->
+        <template id="donation_form_gtc" inherit_id="cms_form.base_form">
```

Comment:
Didn't you see that we have a widget for terms and condition fields in CMS forms ? You could have used that or did you face issues with it?

Reply:
I saw this afterwards, but I didn't test it. I can switch to it if you want ? It might need some refactoring as then the anonymous checkbox won't have the same styles / alignment